test/pecos/xmc/xlinear/test_xlinear.py

```diff
@@ -1168,3 +1168,36 @@ def test_mmap(tmpdir):
         py_pred = py_model.predict(Xt, **kwargs).todense()
         mmap_pred = mmap_model.predict(Xt, **kwargs).todense()
         assert mmap_pred == approx(py_pred, abs=1e-6), f"post_processor:{pp}"
+
+
+def test_mmap_mlmodel(tmpdir):
```

Comment:
I think this test should be put in `https://github.com/amzn/pecos/blob/mainline/test/pecos/xmc/test_xmc.py` since the functionalities added are for the `xmc/base.py` class.

Reply:
Fixed. See latest Revision.